src/status_im/utils/utils.cljs

```diff
@@ -131,3 +131,9 @@
 (defn safe-replace [s m r]
   (when (string? s)
     (string/replace s m r)))
+
+(defn is-not-svg? [some-string]
+  (not (string/includes? some-string ".svg")))
+
+(defn exclude-svg-resources [lst]
+  (filter is-not-svg? lst))
```

Comment:
**not** like it actually matters, but less parenthesis is less parenthesis 
```clojure
(defn is-svg? [some-string]
  (string/includes? some-string ".svg"))
 
(defn exclude-svg-resources [lst]
  (remove is-svg? lst))
```

Reply:
and then probably just `svg?` https://github.com/bbatsov/clojure-style-guide#naming-predicates

Reply:
and likely not a big deal, but shouldn't it be `(string/ends-with? some-string ".svg")`? I should probably stop looking at those lines